spec/features/admin/councils/edit_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Council', type: :feature, js: true do
+  let!(:council) { FactoryBot.create :council }
+
+  context 'anonymous' do
+    it "can't see it" do
+      visit "/admin/councils/#{council.id}"
+      expect(page).to have_text('Rates Rebate 2018/2019')
+      expect(page).to have_text('Log in')
+    end
+  end
+
+  context 'can edit' do
```

Comment:
This test should be the one that shows the Council being edited - with 'fill_in', Submit, etc